backbone/src/main/java/org/researchstack/backbone/model/ConsentSignatureBody.java

```diff
@@ -0,0 +1,45 @@
+package org.researchstack.backbone.model;
```

Comment:
I think the git history was cleared from this file, but I don't imagine it is important.  Also, not sure i'll be able to revive it.